src/XenobladeChroniclesX/Enhancements/ansio/rules.txt

```diff
@@ -0,0 +1,108 @@
+[Definition]
+titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
+name = Anisotropic Filtering
+path = "Xenoblade Chronicles X/Enhancements/Anisotropic Filtering"
+description = Enables anisotropic filtering for ground, wall, and vegetation textures.|Has a minimal performance impact. Made by Cremitif.
+version = 7
+
+[Default]
+#Defined seprately so that they can be change individually if desired
+$anisomount = 16
+$anisorock = 16
+$anisosand = 16
+$anisotree = 16
+$anisolargeplant = 16
+$anisomidplant = 16
+$anisosmallplant = 16
+
+[Preset]
+category = "Aniso: "
```

Comment:
```suggestion
category = "Anisotrophic Filtering Strength"
```
I personally prefer to not have the `:` at the end of the categories. Also, I think this should be fully written out. For the cheat ones it might make more sense, but try to keep these as close to the other graphic packs as possible whenever possible, if you can :)